components/AdminNav/AdminNav.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import {
+  LucideBell,
+  LucideHome,
+  LucideLayoutGrid,
+  LucideUsers,
+} from 'lucide-react';
+import Link from 'next/link';
+import React from 'react';
+import { AdminUserSettings } from '../AdminUserSettings/AdminUserSettings';
+
+/**
+ * The admin nav component.
+ * @returns The rendered admin nav.
+ */
+export const AdminNav = (): React.JSX.Element => {
+  return (
+    <nav className="flex flex-col justify-between h-full pt-2 pb-3 px-3 text-sm text-zinc-400">
```

Comment:
The className for this component needs to be on the parent container.

Reply:
Added to the component